python/nav/web/seeddb/page/netbox/edit.py

```diff
@@ -149,8 +231,19 @@ def get_read_only_variables(request):
             response = test_napalm_connectivity(ip_address, profile)
         else:
             response = None
+            result[profile.id].update(
+                {
+                    "id": profile.id,
+                    "name": profile.name,
+                    "status": False,
+                    "error_message": (
+                        f"Unsupported profile with protocol {profile.protocol}"
```

Comment:
profile.protocol is just an integer, this would be better:

```suggestion
                        f"Connectivity check not supported for profile with protocol {profile.get_protocol_display()}"
```